components/eam/src/chemistry/mozart/mo_chm_diags.F90

```diff
@@ -1500,17 +1501,36 @@ subroutine gaschmmass_diags( lchnk, ncol, vmr, vmr_old, pdeldry, mbar, rdelt, fl
 
     if ( .not. history_gaschmbudget .and. .not. history_gaschmbudget_2D .and. .not. history_gaschmbudget_2D_levels &
          .and. .not. history_UCIgaschmbudget_2D .and. .not. history_UCIgaschmbudget_2D_levels) return
-
-    if (flag(1:4)=='2DCE' .or. flag(1:4)=='2DTE') then 
-       start_index = id_co
-       end_index = id_co
-    elseif (flag(1:4)=='2DCI' .or.flag(1:4)=='2DTI' .or. flag(1:4)=='2DMP') then
-       start_index = id_o3
-       end_index = id_o3
-    else
-       start_index = 1
-       end_index = gas_pcnst
-    endif
+ !kzm ++ 
+    !modification to avoid debug run issue
```

Comment:
"debug run issue" can be rephrased to be clearer, like "...to avoid issues with debug built"

Reply:
@tangq changed